judges/quality-of-service/quality-of-service.rb

```diff
@@ -111,4 +111,16 @@
   end
   f.average_pull_hoc_size = hocs.empty? ? 0 : hocs.sum.to_f / hocs.size
   f.average_pull_files_size = files.empty? ? 0 : files.sum.to_f / files.size
+
+  # Average review time
+  review_times = []
+  Fbe.unmask_repos.each do |repo|
+    Fbe.octo.search_issues(
+      "repo:#{repo} type:pr created:>#{f.since.utc.iso8601[0..9]}"
+    )[:items].each do |pr|
+      review = Fbe.octo.pull_request_reviews(repo, pr[:number]).min_by { |r| r[:submitted_at] }
+      review_times << (review[:submitted_at] - pr[:created_at]).to_i if review
```

Comment:
@Yegorov it seems that here we will calculate the time between pull request creation and review creation. I'm not sure this is what we need. Instead, we need time between "review created" and "PR merged". This would mean the time people spend on reviewing pull requests. Do I miss something?

Reply:
@yegor256 okey, fixed, can you merge https://github.com/zerocracy/fbe/pull/86 please, because test fail